src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC } from 'react';
+import { Person } from '../../types';
+import { PersonItem } from '../PersonItem';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: FC<Props> = ({ people }) => (
+  <table
+    data-cy="peopleTable"
+    className="table is-striped is-hoverable is-narrow is-fullwidth"
+  >
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Sex</th>
+        <th>Born</th>
+        <th>Died</th>
+        <th>Mother</th>
+        <th>Father</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      {people.map(person => (
+        <PersonItem
+          person={person}
+          key={person.slug}
+          mother={person.mother}
+          father={person.father}
+          motherName={person.motherName}
+          fatherName={person.fatherName}
```

Comment:
will be better to pass whole person object
probably further we need some other field from the person, and it will be easier to maintain it if we'll pass the whole object in `PersonItem`